Add minigrant application to resources

Islast opened the pull request:
This branch has minigrant application, but the structural covariance analyses blog is gone. Also have added jekyll stuff to beginning of minigrant application

Islast committed:
Isla's first blog post

KirstieJane committed:
Added author line

KirstieJane committed:
updated authors

KirstieJane committed:
no really updated authors this time

Islast committed:
Update authors.yml

Islast committed:
Update authors.yml

Islast committed:
Add files via upload

Islast committed:
Update 2017-05-12-Hello-Earthling.md

Islast committed:
add Brain Networks In Python blog post

Islast committed:
Merge branch 'master' of https://github.com/WhitakerLab/whitakerlab.github.io

Islast committed:
Changed silly name

Islast committed:
edit blog post

Islast committed:
Merge remote-tracking branch 'upstream/master'

Islast committed:
add minigrant application to resources

Islast committed:
add jekyll front matter to minigrant resource

Islast committed:
Merge remote-tracking branch 'upstream/master'

Islast committed:
remove blog from branch

KirstieJane committed:
Adds Isla as author to blog

This commit includes Kirstie's edits to the mozilla mini grant
resource post. It renamed the file so it has the correct naming
style, updates the date (because it's taken Kirstie a while
to actually review this pull!) and adds Isla as the author of
the post.

KirstieJane committed:
Delete images associated with blog post

These images were left over from a different branch, so I'm
deleting them here. They can absolutely be added back in with the
blog post!

KirstieJane reviewed (dismissed):
Thanks @Islast! I made a couple of changes - removed two of the files that were left over from the blog post branch, added you as an author and updated the date. I also re-named the file because I *think* jekyll requires the posts to start with a date.

Ready to merge though, and since you're on holiday I'm going to do it! 🙌

KirstieJane committed:
Fixes 401 error for link to Bullmore & Sporns paper

KirstieJane committed:
Fix (again) link to Bullmore & Sporns paper

This is really annoying - the link to the Nature Reviews
Neuroscience page is giving a 401 error which means the travis
build is failing :sad: Fingers crossed this new link to pubmed
isn't a problem.

KirstieJane reviewed (approved):
Fixed the travis build failure for a silly closed access link to the NRN Bullmore & Sporns paper.

KirstieJane commented:
Travis is only failing because it can't find the new blog post! I'm confident it will pass when I click merge. HERE WE GO 💯 🎉 🌮  Thank you @Islast!
